src/awsS3/helperFunctions.ts

```diff
@@ -87,3 +87,31 @@ export const postToDatabase = async (fileParams: any, epk: any, endpoint: string
   let data = await res.json()
   return data
 }
+
+
+//********** STEP 3 FOR FILM TEAM & FILM STILLS **********//
+
+export const postFilmMemberToDatabase = async (fileParams: any, filmMember: any, endpoint: string) => {
+  const cookie: any = Cookies.get('csrf-token')
+
+  let usersPostOptions: any = {
+    method: 'POST',
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      "X-Requested-With": "XMLHttpRequest",
+      "X-CSRF-Token": cookie
+    },
+    credentials: 'include',
+    body: JSON.stringify({
+      film_fam_id: filmMember,
```

Comment:
Let's leave this for now, but create an issue for refactoring the helper functions to dry up the code